### PR TITLE
Update Docs and test for testing ansible version

### PR DESCRIPTION
**Pull request by detiber**

### CODE REVIEW

**Comment by detiber** [comment]
aos-ci-test

File: roles/openshift_facts/tasks/main.yml [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-  assert:
-    that:
-    - ansible_version | version_compare('1.8.0', 'ge')
-    - ansible_version | version_compare('1.9.0', 'ne')
-    - ansible_version | version_compare('1.9.0.1', 'ne')
-    
+- name: Verify Ansible version is greater than or equal to 1.9.4 and less than 2.0
+  fail:
+    when:
+    - ansible_version | version_compare('1.9.4', 'lt')
```
Comment (abutcher) [inline roles/openshift_facts/tasks/main.yml]: Should this be `le`?

Comment (detiber) [inline roles/openshift_facts/tasks/main.yml]: @abutcher The assert to fail change probably makes the diff a bit more confusing that it should be.

basically trying to enforce version 1.9.4 or greater but less than 2.0, so only valid match currently is 1.9.4 hence the less than.

Comment (abutcher) [inline roles/openshift_facts/tasks/main.yml]: Haha I had that so backwards

**Comment by abutcher** [comment]
aos-ci-test

**Comment by detiber** [comment]
aos-ci-test

**Comment by detiber** [comment]
aos-ci-test

File: roles/openshift_facts/tasks/main.yml [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
-  assert:
-    that:
-    - ansible_version | version_compare('1.8.0', 'ge')
-    - ansible_version | version_compare('1.9.0', 'ne')
-    - ansible_version | version_compare('1.9.0.1', 'ne')
-    
+- name: Verify Ansible version is greater than or equal to 1.9.4 and less than 2.0
+  fail:
+    msg: "Unsupported ansible version: {{ ansible_version }} found"
+    when: ansible_version | version_compare('1.9.4', 'lt') or ansible_version | version_compare('2.0', 'ge')
```
Comment (abutcher) [inline roles/openshift_facts/tasks/main.yml]: `when:` is indented two extra spaces also I think we could use `ansible_version.full`

```
failed: [10.8.52.230] => {"failed": true}
msg: Unsupported ansible version: {'major': 1, 'full': '1.9.4', 'string': '1.9.4\n  configured module search path = None', 'minor': 9, 'revision': 4} found
```

Comment (detiber) [inline roles/openshift_facts/tasks/main.yml]: @abutcher thanks! haven't had a chance to dig that deeply into it yet.

**Comment by detiber** [comment]
aos-ci-test

**Comment by detiber** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

**Comment by detiber** [comment]
@arilivigni I thought you fixed my perms? :smile:

**Comment by arilivigni** [comment]
I did 
screenshot from 2016-01-25 18-01-21

**Comment by detiber** [comment]
@brenton ptal
@twiest heads up

**Comment by detiber** [comment]
@arilivigni I'm just giving you a hard time :)

**Comment by arilivigni** [comment]
@detiber I know I just don't know why you can't trigger it.  Any ideas?

**Comment by arilivigni** [comment]
@detiber Actually you triggered and then @abutcher did I saw two runs 20 minutes apart.

**Comment by detiber** [comment]
@arilivigni no idea, maybe jenkins is mad at me for all the times I've cursed him under my breath?

**Comment by arilivigni** [comment]
Success > Console Output#8​5 Jan 25, 2016 5:41 PM 32 KBPR #1253: ​Update ​Docs ​and ​test ​for ​te...
Success > Console Output#8​4 Jan 25, 2016 5:21 PM 32 KBPR #1253: ​Update ​Docs ​and ​test ​for ​te...

**Comment by abutcher** [comment]
I just didn't see one in progress from the github side.

**Comment by detiber** [comment]
same here, I saw @abutcher's comment with an age of 3 minutes ago and only 1 check showing.

**Comment by arilivigni** [comment]
@detiber and @abutcher as you can see what I pasted they ran 20 minutes apart, but maybe it didn't refresh in github, I have had to refresh before to get an updated status.  You may want to go directly to the Jenkins jobs next time to check.  At least we know @detiber is added to the admin list the timing may be a separate issue.

**Comment by brenton** [comment]
@detiber, this makes sense to me.  Merging...

**Pull request merged**